README.md

```diff
@@ -1,471 +1,187 @@
-     context: mauro-data-mapper
-     args:
-       ADDITIONAL_PLUGINS: "uk.ac.ox.softeng.maurodatamapper.plugins:mdm-plugin-excel:3.+"
+```bash
+docker compose build
 ```
 
-This will add the latest minor version of the Excel plugin.
-
-### Multiple Instances
-
-If running multiple docker-compose instances then they will all make use of the same initial images, therefore you only need to run the `./make`
-script once per server.
+This will create two images:
 
-### SSH Firewalled Servers
+- `postgres` - For running the Postgres database used by Mauro Data Mapper.
+- `mauro-data-mapper` - Runs an Apache Tomcat web server for hosting the Mauro Data Mapper web applications, including the customised NHS Data Dictionary Orchestrator.
 
-Some servers have the 22 SSH port firewalled for external connections. If this is the case you can change the `base_images/sdk_base/ssh/config` file,
-
-* comment out the `Hostname` field thats currently active * uncomment both commented out `Hostname` and `Port` fields, this will allow git to work using the 443 port which
-  will not be blocked.
-
----
-
-## Run Environment
-
-By adding variables to the `<service>.environment` section of the docker-compose.yml file you can pass them into the container as environment variables. These will override
-any existing configuration variables which are used by default. Any defaults and normally used environment variables can be found in the relevant service's Dockerfile at
-the `ENV` command.
-
-### postgres service
-
-* `POSTGRES_PASSWORD` - This sets the postgres user password for the service, as per the documentation at
-  [Postgres Docker Hub](https://hub.docker.com/_/postgres), it must be set for a docker postgres container. We have set a default but you can override if desired. If you do
-  override it, you will also need to change the `PGPASSWORD` env variable in the mauro-data-mapper section.
-* `DATABASE_USERNAME` - This is the username which will be created inside the Postgres instance to own the database which the MDM service will use. The username is also used
-  by the MDM service to connect to the postgres instance, therefore if you change this you *MUST* also supply it in the environment args for the MDM service
-* `DATABASE_PASSWORD` - This is the password set for the `DATABASE_USERNAME`. It is the password used by the MDM service to connect to this postgres container.
-
-### mauro-data-mapper service
-
-Any grails configuration property found in any of the plugin.yml or application.yml files can be overridden through environment variables. They simply need to be provided in
-the "dot notation" form rather than the "YML new line" format.
-
-e.g. application.yml
-
-```yml
-database:
-  host: localhost
-```
-
-would be overridden by docker-compose.yml
-
-```yml
-services:
-  mauro-data-mapper:
-    environment:
-        database.host: another-host
+Once the images are ready, run the containers in detached mode as follows:
 
+```bash
+docker compose up -d
 ```
 
```

Comment:
It is possible to run `docker compose up -d` without needing a `docker compose build`.  I've not tried this but it should also work for updating.

Reply:
I've updated to include this detail, thanks 👍